update.php

```diff
@@ -1451,14 +1523,17 @@ function update_1163() {
 	$r = q("ALTER TABLE `item` ADD `network` char(32) NOT NULL");
 
 	set_config('system', 'maintenance', 0);
-	if(!$r)
-		return UPDATE_FAILED;
 
-	return UPDATE_SUCCESS;
+	if ($r) {
+		return UPDATE_SUCCESS;
+	}
+
+	return UPDATE_FAILED;
 }
 function update_1164() {
 	set_config('system', 'maintenance', 1);
 
+	/// @TODO If one update fails, should it continue?
```

Comment:
I wouldn't invest much time into this update functionality. Since a long time we are using another functionality. And even if the database update scripts here do fail, there is always the dbstructure script.